tools/serverpod_cli/lib/src/create/database_setup.dart

```diff
@@ -23,12 +24,38 @@ class DatabaseSetup {
       projectName: name,
     );
 
-    var migration = await generator.createMigration(
-      force: false,
-      config: config,
-    );
+    MigrationVersion? migration;
+    try {
+      migration = await generator.createMigration(
+        force: false,
+        config: config,
+      );
+    } on MigrationVersionLoadException catch (e) {
+      log.error(
+        'Unable to determine latest database definition due to a corrupted '
+        'migration. Please re-create or remove the migration version and try '
+        'again. Migration version: "${e.versionName}".',
```

Comment:
Question: In what scenario would it help to rerun?

Reply:
Basically all of these should never happen when creating a migration. Perhaps a single catch all message is better, maybe the one provided bellow with a link to the documentation.